lxd/instance_post.go

```diff
@@ -618,7 +619,7 @@ func instancePostMigration(s *state.State, inst instance.Instance, newName strin
 	return nil
 }
 
-// Move a non-ceph container to another cluster node.
+// Move an instnace to another cluster node. Source and target members must be online.
```

Comment:
Typo of `instance` and the non-ceph part in the previous comment was important please retain.

Reply:
Yep, thanks.